cmd/juju/application/bundle_test.go

```diff
@@ -2057,6 +2057,69 @@ relations:
 		})
 }
 
+func (s *BundleDeployCharmStoreSuite) TestDeployBundlePassesSequences(c *gc.C) {
+	testcharms.UploadCharm(c, s.client, "xenial/django-42", "dummy")
+
+	// Deploy another django app with two units, this will bump the sequences
+	// for machines and the django application. Then remove them both.
+	app := s.Factory.MakeApplication(c, &factory.ApplicationParams{
+		Name: "django"})
+	u1 := s.Factory.MakeUnit(c, &factory.UnitParams{
+		Application: app,
+	})
+	u2 := s.Factory.MakeUnit(c, &factory.UnitParams{
+		Application: app,
+	})
+	var machines []*state.Machine
+	var ids []string
+	destroyUnitsMachine := func(u *state.Unit) {
+		id, err := u.AssignedMachineId()
+		c.Assert(err, jc.ErrorIsNil)
+		ids = append(ids, id)
+		m, err := s.State.Machine(id)
+		c.Assert(err, jc.ErrorIsNil)
+		machines = append(machines, m)
+		c.Assert(m.ForceDestroy(), jc.ErrorIsNil)
+	}
+	// Tear them down again. This is somewhat convoluted, but it is what we need
```

Comment:
I don't see where "again" come into it. We destroy app and each unit/machine once IIANM.